server/src/entity/InventoryItem.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  ManyToMany,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import { MenuItem } from './MenuItem';
+
+export enum itemCategory {
+  UNKNOWN = 'Unknown',
+  VEGETABLE = 'Vegetable',
+  MEAT = 'Meat',
+  DAILY = 'Daily',
+  FRUIT = 'Fruit',
+  OTHERs = 'Others',
+}
+
+@Entity()
+export class InventoryItem extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  public id: string;
+
+  @Column({
+    length: 50,
+  })
+  public ingredientName: string;
```

Comment:
Why does it need to be ingredientName and not just name?

Reply:
because there are other names?